README.md

```diff
@@ -259,3 +259,17 @@ This will run the `detect` phase of NAB on the data files specified in the above
 JSON file. Note that scoring and normalization are not supported with this
 option. Note also that you may see warning messages regarding the lack of labels
 for other files. You can ignore these warnings.
+
+##### Parameter Optimization on NAB
+
+You can run parameter optimization using your own framework or the framework provided by [htm.core](https://github.com/htm-community/htm.core). As of now, this is only enabled for the htm.core detector, but the same can be done for any detector with low effort (see #792 for details).
+**Requirements**
++ Docker Desktop
++ `pip install docker`
+
+**Usage**
+1. Set `use_optimization = True` in the htm.core detector settings.
```

Comment:
Mh NAB does not offer the possibility to add cmd line args that are handed over to the detectors, so this would require a change in the NAB code. Additionally, the detector is controlled by setting bools such as useSpatial directly in the detector itself, so I think this is inline with the current design.

Reply:
true, not worth changing that :+1: